modules/local_storage.js

```diff
@@ -0,0 +1,16 @@
+const formStorage = {
+  title: '',
+  author: '',
+};
+
+export function populateStorage(title, author) {
+  formStorage.title = title;
+  formStorage.author = author;
+  const storeData = JSON.stringify(formStorage);
+  localStorage.setItem('data', storeData);
+}
+
+export function populateNewForm() {
+  const storeData = JSON.parse(localStorage.getItem('data'));
+  return [storeData.title, storeData.author];
+}
```

Comment:
- [ ] Same here, Please, It's one of the project requirements you use `arrow` syntax for functions instead of JavaScript Function Expressions because in this project you learn how to use ES6 new features. Kindly make this :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)